includes/order_details.php

```diff
@@ -43,6 +43,26 @@ function edd_quaderno_add_payment_meta($payment_id) {
 }
 add_action('edd_view_order_details_payment_meta_before', 'edd_quaderno_add_payment_meta', 100);
 
+/**
+* Call edd_get_order_meta with fallback to 'payment_meta'
+*
+* @since  1.26.0
+* @param  mixed $order_id
+* @param  string $key
+* @return @return mixed|void
+*/
+function edd_quaderno_get_order_meta($order_id, $key) {
+  $value = edd_get_order_meta( $order_id, $key, true );
+  if ( empty ( $value ) ) {
+    $payment_meta = edd_get_order_meta( $order_id, 'payment_meta', true );
```

Comment:
This is how you can get a specific payment meta:

```
$payment = new EDD_Payment( $order_id );
$payment_meta = $payment->get_meta( $key );
```

Reply:
Updated.
Still not 100% sure though. 
Please check it.